packages/generator-honeycomb/app/templates/_Dockerfile

```diff
@@ -1,28 +1,33 @@
-FROM node:latest
+FROM node:alpine
 
 # Install Yarn
-RUN curl -o- -L https://yarnpkg.com/install.sh | bash
+RUN apk add --no-cache --virtual .build-deps tar curl bash gnupg \
+  && curl -o- -L https://yarnpkg.com/install.sh | bash \
+  && apk del .build-deps
 ENV PATH /root/.yarn/bin:$PATH
 
 # Create app directory
 RUN mkdir -p /code/<%= dir %>
 WORKDIR /code/<%= dir %>
 
-# Set environment variable
-ENV NODE_ENV production
-
 # Install app dependencies
 COPY package.json /code/<%= dir %>/
-RUN yarn --pure-lockfile
+RUN apk add --no-cache --virtual .app-deps python make g++ \
+  && yarn --pure-lockfile \
+  && yarn cache clean \
+  && apk del .app-deps
 
 # Bundle app source
 COPY . /code/<%= dir %>
 
 # Build
 RUN yarn run build
```

Comment:
@chrishelgert At the moment I think option 2 is okay, but it makes the image 3 times bigger. 😳